packages/synapse-react-client/src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -208,71 +213,85 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
-              className="EntityFinder__Search__Input"
-              type="search"
-              placeholder="Search by name, Wiki contents, or Synapse ID"
-              value={searchInput}
-              onChange={event => {
-                setSearchInput(event.target.value)
-              }}
-              onKeyDown={(event: any) => {
-                if (event.key === 'Enter') {
-                  if (event.target.value === '') {
+            ) : (
+              <>
+                <Button
+                  variant="outlined"
+                  color="primary"
+                  onClick={() => {
+                    setSearchActive(false)
+                    setSearchInput('')
                     setSearchTerms(undefined)
-                  } else {
-                    setSearchTerms(event.target.value.trim().split(' '))
-                  }
-                }
-              }}
-            />
-            {searchInput && (
-              <span
-                className="ClearSearchIcon"
-                onClick={() => {
-                  setSearchInput('')
-                  setSearchTerms(undefined)
-                }}
-              >
-                <IconSvg icon="close" label="Clear Search" />
-              </span>
+                  }}
+                  startIcon={<ArrowBackOutlinedIcon />}
+                  sx={{ flexShrink: 0, height: '100%', mr: 1 }}
+                >
+                  Back to Browse
+                </Button>
+                <TextField
+                  autoFocus
+                  fullWidth
+                  placeholder="Search by name, Wiki contents, or Synapse ID"
+                  value={searchInput}
+                  onChange={event => {
+                    setSearchInput(event.target.value)
+                  }}
+                  onKeyDown={(event: any) => {
+                    if (event.key === 'Enter') {
+                      if (event.target.value === '') {
+                        setSearchTerms(undefined)
+                      } else {
+                        setSearchTerms(event.target.value.trim().split(' '))
+                      }
+                    }
+                  }}
+                  InputProps={{
+                    startAdornment: (
+                      <InputAdornment position="start">
+                        <SearchIcon />
+                      </InputAdornment>
+                    ),
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <IconButton
+                          onClick={() => {
+                            setSearchInput('')
+                            setSearchTerms(undefined)
+                          }}
+                          aria-label="Clear Search"
+                          disabled={!searchInput}
+                        >
+                          <ClearIcon />
+                        </IconButton>
+                      </InputAdornment>
```

Comment:
Rather than hiding/showing an "X" in the search bar depending on whether the user has entered a search term, always show the "X" in the search bar, but disable it when the user hasn't entered a search term.